transformers/test_data/pot_drip.go

```diff
@@ -0,0 +1,48 @@
+package test_data
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"math/rand"
+)
+
+var rawPotDripLog = types.Log{
+	Address: common.HexToAddress(PotAddress()),
+	Topics: []common.Hash{
+		common.HexToHash(constants.PotDripSignature()),
+		common.HexToHash("0x00000000000000000000000087e76b0a50efc20259cafe0530f75ae0e816aaf2"),
```

Comment:
I like this approach of not including more topics than necessary in our test data. Now that I think about it, would it make sense to omit the `Data` field on line 19 for converters that don't need log data?

Reply:
tbh I removed them because I saw some error when they were included. But I'm not able to reproduce that and thinking maybe I copy/pasted the empty topics wrong. I'm inclined to re-add them and keep everything to accurately simulate what we'd expect over the RPC